src/repr/src/global_id.rs

```diff
@@ -62,10 +64,15 @@ pub enum GlobalId {
     Explain,
 }
 
+static_assertions::assert_eq_size!(GlobalId, [u8; 16]);
```

Comment:
Could you add a comment explaining why we have this assert?

Reply:
Done